go-controller/pkg/ovn/obj_retry.go

```diff
@@ -0,0 +1,140 @@
+package ovn
+
+import (
+	"reflect"
+	"sync"
+	"time"
+
+	"k8s.io/klog/v2"
+)
+
+// initRetryObj tracks a failed object to potentially retry later
+// initially it is marked as skipped for retry loop (ignore = true)
+func (oc *Controller) initRetryObj(obj interface{}, lock *sync.Mutex, retryObjs map[string]interface{}, key string) {
+	var entry *retryObjEntry
+	(*lock).Lock()
+	defer (*lock).Unlock()
+	v := reflect.ValueOf(retryObjs)
+	if v.MapIndex(reflect.ValueOf(key)).IsValid() {
+		entry = v.MapIndex(reflect.ValueOf(key)).Interface().(*retryObjEntry)
```

Comment:
if the value that the map holds is always a retryObjEntry, it doesn't need to be an interface. Only the retryObjEntry->newObj (etc) needs to be an interface because it can hold different kapi types. Change the map to map[string]*retryObjEntry